snapcraft.yaml

```diff
@@ -77,7 +81,14 @@ parts:
       - setuptools
       - tqdm
       - pipe
+      - wheel
 
     override-build: |
+      pip3 install --user dephell[full]
+      $HOME/.local/bin/dephell deps convert \
+          --from-path pyproject.toml \
+          --from-format poetry \
```

Comment:
ocrmypdf uses [setuptools pyproject.toml](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html) not poetry. Are you sure `--from-format poetry` is correct?

Reply:
Yeah, that should be wrong, I'll check it out.

Reply:
@jbarlow83 Revision 22531be should fix the problem, please review.